source/documentation/values/calculations.html.md.erb

```diff
@@ -161,42 +171,55 @@ CSS to unitless numbers:
   @debug calc(-infinity) < math.$min-number  // true
 <% end %>
 
-## `min()` and `max()`
+## [Calculation Functions]
```

Comment:
This needs an `impl_status` that indicates that the new functions were added in 1.64.0.

Reply:
Done!